<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Differential Binding Region (MD vs MV) | SLX-16110-16111 Run Report (ChIP-Seq)</title>
  <meta name="description" content="This is a report for Irving Aye’s `r params$seq_type’ results" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Differential Binding Region (MD vs MV) | SLX-16110-16111 Run Report (ChIP-Seq)" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a report for Irving Aye’s `r params$seq_type’ results" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Differential Binding Region (MD vs MV) | SLX-16110-16111 Run Report (ChIP-Seq)" />
  
  <meta name="twitter:description" content="This is a report for Irving Aye’s `r params$seq_type’ results" />
  



<meta name="date" content="2021-02-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="diffbind.html"/>
<link rel="next" href="chipseqspikeinfree.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-3.5.1/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.17/datatables.js"></script>
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<script src="libs/jszip-1.10.20/jszip.min.js"></script>
<link href="libs/dt-ext-buttons-1.10.20/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="libs/dt-ext-buttons-1.10.20/js/dataTables.buttons.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.flash.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.html5.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.colVis.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.print.min.js"></script>
<link href="libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Run Report</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Samples</a></li>
<li class="chapter" data-level="2" data-path="mapping.html"><a href="mapping.html"><i class="fa fa-check"></i><b>2</b> Mapping</a>
<ul>
<li class="chapter" data-level="2.1" data-path="mapping.html"><a href="mapping.html#table-of-mapping-statistic"><i class="fa fa-check"></i><b>2.1</b> Table of mapping statistic</a></li>
<li class="chapter" data-level="2.2" data-path="mapping.html"><a href="mapping.html#boxplots-of-mapping-staststic"><i class="fa fa-check"></i><b>2.2</b> Boxplots of mapping staststic</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="mapping.html"><a href="mapping.html#no.-of-sequencing-reads"><i class="fa fa-check"></i><b>2.2.1</b> No. of sequencing reads</a></li>
<li class="chapter" data-level="2.2.2" data-path="mapping.html"><a href="mapping.html#no.-of-uniquely-mapped-reads"><i class="fa fa-check"></i><b>2.2.2</b> No. of uniquely mapped reads</a></li>
<li class="chapter" data-level="2.2.3" data-path="mapping.html"><a href="mapping.html#unique-mapping-rate"><i class="fa fa-check"></i><b>2.2.3</b> Unique mapping rate</a></li>
<li class="chapter" data-level="2.2.4" data-path="mapping.html"><a href="mapping.html#multi-mapping-rate"><i class="fa fa-check"></i><b>2.2.4</b> Multi mapping rate</a></li>
<li class="chapter" data-level="2.2.5" data-path="mapping.html"><a href="mapping.html#total-mapping-rate"><i class="fa fa-check"></i><b>2.2.5</b> Total mapping rate</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="mapping.html"><a href="mapping.html#bigwig-files"><i class="fa fa-check"></i><b>2.3</b> BigWig files</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="calling.html"><a href="calling.html"><i class="fa fa-check"></i><b>3</b> Peak Calling</a>
<ul>
<li class="chapter" data-level="3.1" data-path="calling.html"><a href="calling.html#no.-of-peaks"><i class="fa fa-check"></i><b>3.1</b> No. of peaks</a></li>
<li class="chapter" data-level="3.2" data-path="calling.html"><a href="calling.html#no.-of-peaks-by-the-consensus-level"><i class="fa fa-check"></i><b>3.2</b> No. of peaks by the consensus level</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="calling.html"><a href="calling.html#all-samples-n22"><i class="fa fa-check"></i><b>3.2.1</b> All samples (n=22)</a></li>
<li class="chapter" data-level="3.2.2" data-path="calling.html"><a href="calling.html#fv-n5"><i class="fa fa-check"></i><b>3.2.2</b> FV (n=5)</a></li>
<li class="chapter" data-level="3.2.3" data-path="calling.html"><a href="calling.html#fd-n5"><i class="fa fa-check"></i><b>3.2.3</b> FD (n=5)</a></li>
<li class="chapter" data-level="3.2.4" data-path="calling.html"><a href="calling.html#mv-n6"><i class="fa fa-check"></i><b>3.2.4</b> MV (n=6)</a></li>
<li class="chapter" data-level="3.2.5" data-path="calling.html"><a href="calling.html#md-n6"><i class="fa fa-check"></i><b>3.2.5</b> MD (n=6)</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="calling.html"><a href="calling.html#consensus_peak"><i class="fa fa-check"></i><b>3.3</b> Consensus peaks across factors of interests</a></li>
<li class="chapter" data-level="3.4" data-path="calling.html"><a href="calling.html#fraction-of-read-in-peaks-frip"><i class="fa fa-check"></i><b>3.4</b> Fraction of Read in Peaks (FRiP)</a></li>
<li class="chapter" data-level="3.5" data-path="calling.html"><a href="calling.html#heatmap"><i class="fa fa-check"></i><b>3.5</b> Heatmap</a></li>
<li class="chapter" data-level="3.6" data-path="calling.html"><a href="calling.html#pca"><i class="fa fa-check"></i><b>3.6</b> PCA</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="peak-annotations.html"><a href="peak-annotations.html"><i class="fa fa-check"></i><b>4</b> Peak Annotations</a>
<ul>
<li class="chapter" data-level="4.1" data-path="peak-annotations.html"><a href="peak-annotations.html#no.-of-peaks-by-group"><i class="fa fa-check"></i><b>4.1</b> No. of peaks by group</a></li>
<li class="chapter" data-level="4.2" data-path="peak-annotations.html"><a href="peak-annotations.html#no.-of-peaks-by-chromosome"><i class="fa fa-check"></i><b>4.2</b> No. of peaks by chromosome</a></li>
<li class="chapter" data-level="4.3" data-path="peak-annotations.html"><a href="peak-annotations.html#no.-of-genes-enrichred-in-peaks"><i class="fa fa-check"></i><b>4.3</b> No. of genes enrichred in peaks</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="peak-annotations.html"><a href="peak-annotations.html#non-intergenic-peaks-only"><i class="fa fa-check"></i><b>4.3.1</b> Non-intergenic peaks only</a></li>
<li class="chapter" data-level="4.3.2" data-path="peak-annotations.html"><a href="peak-annotations.html#peaks-in-tss-2kb"><i class="fa fa-check"></i><b>4.3.2</b> Peaks in TSS (&lt;2kb)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="peak-annotations.html"><a href="peak-annotations.html#genomic-feature-of-peaks"><i class="fa fa-check"></i><b>4.4</b> Genomic feature of peaks</a></li>
<li class="chapter" data-level="4.5" data-path="peak-annotations.html"><a href="peak-annotations.html#distance-to-tss"><i class="fa fa-check"></i><b>4.5</b> Distance to TSS</a></li>
<li class="chapter" data-level="4.6" data-path="peak-annotations.html"><a href="peak-annotations.html#peak-frequency-over-tss"><i class="fa fa-check"></i><b>4.6</b> Peak frequency over TSS</a></li>
<li class="chapter" data-level="4.7" data-path="peak-annotations.html"><a href="peak-annotations.html#peak-heatmap-over-tss"><i class="fa fa-check"></i><b>4.7</b> Peak heatmap over TSS</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="diffbind.html"><a href="diffbind.html"><i class="fa fa-check"></i><b>5</b> Differential Binding Region</a>
<ul>
<li class="chapter" data-level="5.1" data-path="diffbind.html"><a href="diffbind.html#no.-of-dbr"><i class="fa fa-check"></i><b>5.1</b> No. of DBR</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="diffbind.html"><a href="diffbind.html#adjutedd-p-val0.05"><i class="fa fa-check"></i><b>5.1.1</b> adjutedd P-val&lt;0.05</a></li>
<li class="chapter" data-level="5.1.2" data-path="diffbind.html"><a href="diffbind.html#adjutedd-p-val0.01"><i class="fa fa-check"></i><b>5.1.2</b> adjutedd P-val&lt;0.01:</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="diffbind.html"><a href="diffbind.html#dbr-from-deseq2"><i class="fa fa-check"></i><b>5.2</b> DBR from DESeq2</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="diffbind.html"><a href="diffbind.html#normalisation-factors"><i class="fa fa-check"></i><b>5.2.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="5.2.2" data-path="diffbind.html"><a href="diffbind.html#male-md-vs-mv"><i class="fa fa-check"></i><b>5.2.2</b> Male (MD vs MV)</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="diffbind.html"><a href="diffbind.html#dbr-from-edger"><i class="fa fa-check"></i><b>5.3</b> DBR from edgeR</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="diffbind.html"><a href="diffbind.html#normalisation-factors-1"><i class="fa fa-check"></i><b>5.3.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="5.3.2" data-path="diffbind.html"><a href="diffbind.html#male-md-vs-mv-1"><i class="fa fa-check"></i><b>5.3.2</b> Male (MD vs MV)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html"><i class="fa fa-check"></i><b>6</b> Differential Binding Region (MD vs MV)</a>
<ul>
<li class="chapter" data-level="6.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#no.-of-dbr-1"><i class="fa fa-check"></i><b>6.1</b> No. of DBR</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#adjutedd-p-val0.05-1"><i class="fa fa-check"></i><b>6.1.1</b> adjutedd P-val&lt;0.05</a></li>
<li class="chapter" data-level="6.1.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#adjutedd-p-val0.01-1"><i class="fa fa-check"></i><b>6.1.2</b> adjutedd P-val&lt;0.01:</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#dbr-from-deseq2-1"><i class="fa fa-check"></i><b>6.2</b> DBR from DESeq2</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#normalisation-factors-2"><i class="fa fa-check"></i><b>6.2.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="6.2.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#male-md-vs-mv-2"><i class="fa fa-check"></i><b>6.2.2</b> Male (MD vs MV)</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#dbr-from-edger-1"><i class="fa fa-check"></i><b>6.3</b> DBR from edgeR</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#normalisation-factors-3"><i class="fa fa-check"></i><b>6.3.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="6.3.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#male-md-vs-mv-3"><i class="fa fa-check"></i><b>6.3.2</b> Male (MD vs MV)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="chipseqspikeinfree.html"><a href="chipseqspikeinfree.html"><i class="fa fa-check"></i><b>7</b> ChIPseqSpikeInFree</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a>
<ul>
<li class="chapter" data-level="7.1" data-path="references.html"><a href="references.html#sessioninfo"><i class="fa fa-check"></i><b>7.1</b> sessionInfo</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">SLX-16110-16111 Run Report (ChIP-Seq)</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="diffbind-md-vs-mv-n-10" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Differential Binding Region (MD vs MV)</h1>
<blockquote>
<p>Based on 71258 consensus peaks (see <a href="calling.html#consensus_peak">here</a>)</p>
<p>Software package used: <a href="https://bioconductor.org/packages/release/bioc/html/DiffBind.html">DiffBind</a></p>
<p>Both M2D and M2V were removed from the analysis</p>
</blockquote>
<blockquote>
<p><a href="https://bioconductor.org/packages/3.12/bioc/vignettes/DiffBind/inst/doc/DiffBind.pdf#page=38"><code>Background normalisation</code></a> method using <code>DiffBind3</code></p>
</blockquote>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="diffbind-md-vs-mv-n-10.html#cb13-1" aria-hidden="true" tabindex="-1"></a>li.diff</span></code></pre></div>
<pre><code>## $bg
## $bg$female
## 10 Samples, 71258 sites in matrix:
##         ID Tissue Factor Condition Replicate    Reads
## 1  F1V-K27      F     FV         V         1 14408092
## 2  F1D-K27      F     FD         D         1 14315340
## 3  F2V-K27      F     FV         V         2  9053481
## 4  F2D-K27      F     FD         D         2  8275061
## 5  F3V-K27      F     FV         V         3 10783204
## 6  F3D-K27      F     FD         D         3 10635286
## 7  F4V-K27      F     FV         V         4  7783770
## 8  F4D-K27      F     FD         D         4  8817488
## 9  F5V-K27      F     FV         V         5 10164136
## 10 F5D-K27      F     FD         D         5 15060952
## 
## Design: [~Factor] | 1 Contrast:
##   Factor Group Samples Group2 Samples2 DB.edgeR DB.DESeq2
## 1 Factor    FD       5     FV        5        0         0
## 
## $bg$male
## 12 Samples, 71258 sites in matrix:
##         ID Tissue Factor Condition Replicate    Reads
## 1  M1V-K27      M     MV         V         1  7845159
## 2  M1D-K27      M     MD         D         1  6258045
## 3  M2V-K27      M     MV         V         2  7621276
## 4  M2D-K27      M     MD         D         2 45666925
## 5  M3V-K27      M     MV         V         3 16276900
## 6  M3D-K27      M     MD         D         3 12004853
## 7  M4V-K27      M     MV         V         4 13044881
## 8  M4D-K27      M     MD         D         4 14754594
## 9  M5V-K27      M     MV         V         5 14302563
## 10 M5D-K27      M     MD         D         5 13953234
## 11 M6V-K27      M     MV         V         6 10103163
## 12 M6D-K27      M     MD         D         6  5879122
## 
## Design: [~Factor] | 1 Contrast:
##   Factor Group Samples Group2 Samples2 DB.edgeR DB.DESeq2
## 1 Factor    MD       6     MV        6        6        33</code></pre>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="diffbind-md-vs-mv-n-10.html#cb15-1" aria-hidden="true" tabindex="-1"></a>li.diff[[<span class="st">&#39;bg&#39;</span>]][[<span class="st">&quot;male-n10&quot;</span>]]<span class="ot">&lt;-</span><span class="fu">dba</span>(dbo.cnt,dbo.cnt<span class="sc">$</span>masks<span class="sc">$</span>M <span class="sc">&amp;</span> <span class="sc">!</span>dbo.cnt<span class="sc">$</span>mask<span class="sc">$</span>Replicate<span class="fl">.2</span>) <span class="co"># isa DBA</span></span>
<span id="cb15-2"><a href="diffbind-md-vs-mv-n-10.html#cb15-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-3"><a href="diffbind-md-vs-mv-n-10.html#cb15-3" aria-hidden="true" tabindex="-1"></a>li.diff[[<span class="st">&#39;bg&#39;</span>]][<span class="st">&#39;male-n10&#39;</span>]<span class="ot">&lt;-</span><span class="fu">lapply</span>(li.diff[[<span class="st">&#39;bg&#39;</span>]][<span class="st">&#39;male-n10&#39;</span>], dba.normalize, <span class="at">method=</span>DBA_ALL_METHODS,<span class="at">normalize=</span>DBA_NORM_NATIVE,<span class="at">library=</span>DBA_LIBSIZE_FULL,<span class="at">background=</span><span class="cn">TRUE</span>)</span>
<span id="cb15-4"><a href="diffbind-md-vs-mv-n-10.html#cb15-4" aria-hidden="true" tabindex="-1"></a>li.diff[[<span class="st">&#39;bg&#39;</span>]][<span class="st">&#39;male-n10&#39;</span>]<span class="ot">&lt;-</span><span class="fu">lapply</span>(li.diff[[<span class="st">&#39;bg&#39;</span>]][<span class="st">&#39;male-n10&#39;</span>], dba.analyze, <span class="at">method=</span>DBA_ALL_METHODS,<span class="at">bBlacklist=</span>F,<span class="at">bGreylist=</span>F)</span>
<span id="cb15-5"><a href="diffbind-md-vs-mv-n-10.html#cb15-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-6"><a href="diffbind-md-vs-mv-n-10.html#cb15-6" aria-hidden="true" tabindex="-1"></a><span class="fu">lapply</span>(li.diff[[<span class="st">&#39;bg&#39;</span>]][<span class="st">&#39;male-n10&#39;</span>], dba.show, <span class="at">bContrasts=</span><span class="cn">TRUE</span>)</span></code></pre></div>
<pre><code>## $`male-n10`
##   Factor Group Samples Group2 Samples2 DB.edgeR DB.DESeq2
## 1 Factor    MD       5     MV        5        4        15</code></pre>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="diffbind-md-vs-mv-n-10.html#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">lapply</span>(li.diff[[<span class="st">&#39;bg&#39;</span>]][<span class="st">&#39;male-n10&#39;</span>], dba.normalize, <span class="at">bRetrieve=</span>T,<span class="at">method=</span>DBA_ALL_METHODS)</span></code></pre></div>
<pre><code>## $`male-n10`
## $`male-n10`$edgeR
## $`male-n10`$edgeR$background
## [1] TRUE
## 
## $`male-n10`$edgeR$norm.method
## [1] &quot;TMM&quot;
## 
## $`male-n10`$edgeR$norm.factors
##  [1] 1.0344810 1.0478794 1.0038036 1.0096211 0.9815742 1.0476543 0.9146446
##  [8] 0.9552479 1.0028985 1.0101653
## 
## $`male-n10`$edgeR$lib.method
## [1] &quot;background&quot;
## 
## $`male-n10`$edgeR$lib.sizes
##  [1]  7843284  6257352 16276097 12004098 13043893 14753138 14301752 13952221
##  [9] 10102444  5878538
## 
## $`male-n10`$edgeR$control.subtract
## [1] TRUE
## 
## $`male-n10`$edgeR$filter.value
## [1] 1
## 
## 
## $`male-n10`$DESeq2
## $`male-n10`$DESeq2$background
## [1] TRUE
## 
## $`male-n10`$DESeq2$norm.method
## [1] &quot;RLE&quot;
## 
## $`male-n10`$DESeq2$norm.factors
##   M1V-K27   M1D-K27   M3V-K27   M3D-K27   M4V-K27   M4D-K27   M5V-K27   M5D-K27 
## 0.7523919 0.6071245 1.5260973 1.1285218 1.1962884 1.4409517 1.2072382 1.2366381 
##   M6V-K27   M6D-K27 
## 0.9450624 0.5526180 
## 
## $`male-n10`$DESeq2$lib.method
## [1] &quot;background&quot;
## 
## $`male-n10`$DESeq2$lib.sizes
##  [1]  7843284  6257352 16276097 12004098 13043893 14753138 14301752 13952221
##  [9] 10102444  5878538
## 
## $`male-n10`$DESeq2$control.subtract
## [1] TRUE
## 
## $`male-n10`$DESeq2$filter.value
## [1] 1
## 
## 
## $`male-n10`$background
## $`male-n10`$background$binned
## class: RangedSummarizedExperiment 
## dim: 189187 10 
## metadata(6): spacing width ... param final.ext
## assays(1): counts
## rownames: NULL
## rowData names(0):
## colnames(10): M1V-K27 M1D-K27 ... M6V-K27 M6D-K27
## colData names(4): bam.files totals ext rlen
## 
## $`male-n10`$background$bin.size
## [1] 15000
## 
## $`male-n10`$background$back.calc
## [1] &quot;Background bins&quot;</code></pre>
<div id="no.-of-dbr-1" class="section level2" number="6.1">
<h2><span class="header-section-number">6.1</span> No. of DBR</h2>
<div id="adjutedd-p-val0.05-1" class="section level3" number="6.1.1">
<h3><span class="header-section-number">6.1.1</span> adjutedd P-val&lt;0.05</h3>
<table>
<caption>
<span id="tab:diff-bind-num1-m10">Table 6.1: </span>Number of differential binding regions
</caption>
<thead>
<tr>
<th style="text-align:left;">
Factor
</th>
<th style="text-align:left;">
Group
</th>
<th style="text-align:left;">
Samples
</th>
<th style="text-align:left;">
Group2
</th>
<th style="text-align:left;">
Samples2
</th>
<th style="text-align:left;">
DB.edgeR
</th>
<th style="text-align:left;">
DB.DESeq2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Factor
</td>
<td style="text-align:left;">
MD
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
MV
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
15
</td>
</tr>
</tbody>
</table>
<pre><code>## $`male-n10`
## 2 Samples, 15 sites in matrix:
##    Contrast Direction DB Method Intervals
## 1 MD vs. MV       All DB  edgeR         4
## 2 MD vs. MV       All DB DESeq2        15</code></pre>
<div class="figure" style="text-align: center"><span id="fig:diff-bind-venn1-m10"></span>
<img src="SLX-16110-16111_files/figure-html/diff-bind-venn1-m10-1.png" alt="DBR between edgeR &amp; DESeq2 (MD vs MV)" width="672" />
<p class="caption">
Figure 6.1: DBR between edgeR &amp; DESeq2 (MD vs MV)
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:diff-bind-pca1-m10"></span>
<img src="SLX-16110-16111_files/figure-html/diff-bind-pca1-m10-1.png" alt="PCA based on DBR (MD vs MV)" width="672" />
<p class="caption">
Figure 6.2: PCA based on DBR (MD vs MV)
</p>
</div>
</div>
<div id="adjutedd-p-val0.01-1" class="section level3" number="6.1.2">
<h3><span class="header-section-number">6.1.2</span> adjutedd P-val&lt;0.01:</h3>
<table>
<caption>
<span id="tab:diff-bind-num2-m-m10">Table 6.2: </span>Number of differential binding regions
</caption>
<thead>
<tr>
<th style="text-align:left;">
Factor
</th>
<th style="text-align:left;">
Group
</th>
<th style="text-align:left;">
Samples
</th>
<th style="text-align:left;">
Group2
</th>
<th style="text-align:left;">
Samples2
</th>
<th style="text-align:left;">
DB.edgeR
</th>
<th style="text-align:left;">
DB.DESeq2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Factor
</td>
<td style="text-align:left;">
MD
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
MV
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dbr-from-deseq2-1" class="section level2" number="6.2">
<h2><span class="header-section-number">6.2</span> DBR from DESeq2</h2>
<div id="normalisation-factors-2" class="section level3" number="6.2.1">
<h3><span class="header-section-number">6.2.1</span> Normalisation factors</h3>
<div class="figure" style="text-align: center"><span id="fig:dbr-norm-factor-deseq2-m-m10"></span>
<img src="SLX-16110-16111_files/figure-html/dbr-norm-factor-deseq2-m-m10-1.png" alt="Normalisation factors by group" width="672" />
<p class="caption">
Figure 6.3: Normalisation factors by group
</p>
</div>
</div>
<div id="male-md-vs-mv-2" class="section level3" number="6.2.2">
<h3><span class="header-section-number">6.2.2</span> Male (MD vs MV)</h3>
<div id="htmlwidget-6e38d1255b33db858196" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-6e38d1255b33db858196">{"x":{"filter":"none","extensions":["Buttons"],"caption":"<caption>DBR of MD vs MV (DESeq2; FDR&lt;0.05)<\/caption>","data":[["2","7","1","9","13","2","20","19","4","10","17","1","2","17","3"],[10331278,155313109,22858106,129472056,110950340,217335920,40433510,48338600,6723121,30420253,28247240,39936237,101411159,40915550,16789344],[10331800,155313774,22858404,129472580,110950850,217336666,40433782,48339706,6723841,30420531,28248335,39937221,101412366,40916939,16789790],[523,666,299,525,511,747,273,1107,721,279,1096,985,1208,1390,447],["*","*","*","*","*","*","*","*","*","*","*","*","*","*","*"],[2.42428491478203,3.50413527672415,2.07515906116423,3.11850463994959,2.95087053584097,3.63686320673893,2.26902450222588,4.78251794358734,3.56834155582236,1.88660864176491,4.55112280796226,4.11089486239381,4.7011156892197,4.16835233701663,2.70883648790555],[0,1.14407796588749,0,0.713137614297469,0.074846512940887,1.94942150875597,0,3.82015738887263,1.57962012268441,0,3.56181391948941,2.8078295159722,3.72333255690628,2.90348129931085,0.2922375779757],[3.42428491478203,4.35630722124403,3.07515906116423,3.97548153155628,3.84910462958675,4.39347919746478,3.21381829612634,5.35471253106995,4.37407803209656,2.88660864176491,5.13250392614393,4.78421229187914,5.27859588248746,4.83180028555765,3.56697891186425],[-3.88851597384936,-2.42686395565281,-3.1083235152806,-2.36215011205194,-2.8378186896965,-1.92293326125318,-2.61225744150103,-1.37830060623193,-1.99039837479303,-2.34463578274638,-1.4005864811342,-1.65248869174889,-1.38730918199881,-1.57469832223402,-2.16329587348384],[3.4110034703073e-08,6.33238657984194e-08,6.14515137619494e-07,7.15143502492249e-07,8.85180664959992e-07,1.70602388085819e-06,2.31711545728389e-06,3.29546768008821e-06,3.30460195049665e-06,3.90806090229684e-06,3.9180103233451e-06,5.13299745869066e-06,5.71796310819313e-06,8.09522432494135e-06,8.70019480016669e-06],[0.00225455125595403,0.00225455125595403,0.0126062119219612,0.0126062119219612,0.0126062119219612,0.0202468070807116,0.0235706914809734,0.0253627055540395,0.0253627055540395,0.0253627055540395,0.0253627055540395,0.0304587791700822,0.0313199230034699,0.0411740456075785,0.0413009847423646],["Promoter (1-2kb)","Distal Intergenic","Intron (ENST00000544305.5/2048, intron 3 of 6)","Distal Intergenic","Intron (ENST00000652145.1/ENST00000652145.1, intron 2 of 2)","Promoter (&lt;=1kb)","Distal Intergenic","Intron (ENST00000293261.8/55260, intron 6 of 7)","Distal Intergenic","Distal Intergenic","Promoter (&lt;=1kb)","Distal Intergenic","Intron (ENST00000428343.6/731220, intron 8 of 11)","Distal Intergenic","Distal Intergenic"],[2,7,1,9,13,2,20,19,4,10,17,1,2,17,3],[10330163,155301607,22863159,129476946,110965704,217284025,40685848,48333276,6716174,30434021,28247444,39955112,101421383,40929676,16802651],[10427616,155303018,22863226,129504274,110990579,217336618,40689236,48342991,6717664,30461833,28248289,39969968,101435106,40936390,17011084],[97454,1412,68,27329,24876,52594,3389,9716,1491,27813,846,14857,13724,6715,208434],[1,1,2,1,1,2,2,2,1,1,1,1,2,2,1],["3241","3638","100422914","100506190","104355135","729582","9935","55260","55330","1326","23614","84879","731220","25984","23228"],["ENST00000622018.4","ENST00000468307.1","ENST00000581187.1","ENST00000653047.1","ENST00000440735.2","ENST00000654616.1","ENST00000373313.3","ENST00000600816.1","ENST00000320776.5","ENST00000542547.5","ENST00000583761.1","ENST00000420632.6","ENST00000492238.1","ENST00000584517.1","ENST00000460467.1"],[1115,11502,4822,-4366,-14854,0,255454,3285,6947,-13490,0,-17891,22740,19451,-12861],["ENSG00000115756","ENSG00000186480","ENSG00000264014",null,null,null,"ENSG00000204103","ENSG00000161558","ENSG00000186222","ENSG00000107968","ENSG00000265060","ENSG00000168389","ENSG00000196460","ENSG00000108244","ENSG00000154822"],["HPCAL1","INSIG1","MIR4253","LINC00963","LINC00431","DIRC3","MAFB","TMEM143","BLOC1S4","MAP3K8","PPY2P","MFSD2A","RFX8","KRT23","PLCL2"],["hippocalcin like 1","insulin induced gene 1","microRNA 4253","long intergenic non-protein coding RNA 963","long intergenic non-protein coding RNA 431","disrupted in renal carcinoma 3","MAF bZIP transcription factor B","transmembrane protein 143","biogenesis of lysosomal organelles complex 1 subunit 4","mitogen-activated protein kinase kinase kinase 8","pancreatic polypeptide 2, pseudogene","major facilitator superfamily domain containing 2A","regulatory factor X8","keratin 23","phospholipase C like 2"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Chr<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>width<\/th>\n      <th>strand<\/th>\n      <th>Conc<\/th>\n      <th>Conc_MD<\/th>\n      <th>Conc_MV<\/th>\n      <th>Fold<\/th>\n      <th>p.value<\/th>\n      <th>FDR<\/th>\n      <th>annotation<\/th>\n      <th>geneChr<\/th>\n      <th>geneStart<\/th>\n      <th>geneEnd<\/th>\n      <th>geneLength<\/th>\n      <th>geneStrand<\/th>\n      <th>geneId<\/th>\n      <th>transcriptId<\/th>\n      <th>distanceToTSS<\/th>\n      <th>ENSEMBL<\/th>\n      <th>SYMBOL<\/th>\n      <th>GENENAME<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"scrollX":true,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"className":"dt-right","targets":[1,2,3,5,6,7,8,9,10,12,13,14,15,16,19]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<div class="figure" style="text-align: center"><span id="fig:plotMA-male-deseq2-fig1-m10"></span>
<img src="SLX-16110-16111_files/figure-html/plotMA-male-deseq2-fig1-m10-1.png" alt="MA plot (DESeq2)" width="672" />
<p class="caption">
Figure 6.4: MA plot (DESeq2)
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:plotVol-male-deseq2-fig1-m10"></span>
<img src="SLX-16110-16111_files/figure-html/plotVol-male-deseq2-fig1-m10-1.png" alt="Volcano plot (DESeq2)" width="672" />
<p class="caption">
Figure 6.5: Volcano plot (DESeq2)
</p>
</div>
</div>
</div>
<div id="dbr-from-edger-1" class="section level2" number="6.3">
<h2><span class="header-section-number">6.3</span> DBR from edgeR</h2>
<div id="normalisation-factors-3" class="section level3" number="6.3.1">
<h3><span class="header-section-number">6.3.1</span> Normalisation factors</h3>
<div class="figure" style="text-align: center"><span id="fig:dbr-norm-factor-edger-m10"></span>
<img src="SLX-16110-16111_files/figure-html/dbr-norm-factor-edger-m10-1.png" alt="Normalisation factors by group" width="672" />
<p class="caption">
Figure 6.6: Normalisation factors by group
</p>
</div>
</div>
<div id="male-md-vs-mv-3" class="section level3" number="6.3.2">
<h3><span class="header-section-number">6.3.2</span> Male (MD vs MV)</h3>
<div id="htmlwidget-bc060f03be28c210bd3c" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-bc060f03be28c210bd3c">{"x":{"filter":"none","extensions":["Buttons"],"caption":"<caption>DBR of MD vs MV (edgeR; FDR&lt;0.05)<\/caption>","data":[["2","7","13","1"],[10331278,155313109,110950340,22858106],[10331800,155313774,110950850,22858404],[523,666,511,299],["*","*","*","*"],[2.6041083294419,3.68013938152383,3.06274489938795,2.25858183619478],[0,1.31647741553508,0,0],[3.6041083294419,4.53269973162371,4.00538724595437,3.25858183619478],[-6.5327969258264,-2.98068379126081,-4.20170617616338,-6.19143211206459],[5.55218847183766e-08,3.01071063542842e-07,4.29179287919915e-07,1.62395865092594e-06],[0.00395354684514144,0.0101868565183045,0.0101868565183045,0.0289093059141209],["Promoter (1-2kb)","Distal Intergenic","Intron (ENST00000652145.1/ENST00000652145.1, intron 2 of 2)","Intron (ENST00000544305.5/2048, intron 3 of 6)"],[2,7,13,1],[10330163,155301607,110965704,22863159],[10427616,155303018,110990579,22863226],[97454,1412,24876,68],[1,1,1,2],["3241","3638","104355135","100422914"],["ENST00000622018.4","ENST00000468307.1","ENST00000440735.2","ENST00000581187.1"],[1115,11502,-14854,4822],["ENSG00000115756","ENSG00000186480",null,"ENSG00000264014"],["HPCAL1","INSIG1","LINC00431","MIR4253"],["hippocalcin like 1","insulin induced gene 1","long intergenic non-protein coding RNA 431","microRNA 4253"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Chr<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>width<\/th>\n      <th>strand<\/th>\n      <th>Conc<\/th>\n      <th>Conc_MD<\/th>\n      <th>Conc_MV<\/th>\n      <th>Fold<\/th>\n      <th>p.value<\/th>\n      <th>FDR<\/th>\n      <th>annotation<\/th>\n      <th>geneChr<\/th>\n      <th>geneStart<\/th>\n      <th>geneEnd<\/th>\n      <th>geneLength<\/th>\n      <th>geneStrand<\/th>\n      <th>geneId<\/th>\n      <th>transcriptId<\/th>\n      <th>distanceToTSS<\/th>\n      <th>ENSEMBL<\/th>\n      <th>SYMBOL<\/th>\n      <th>GENENAME<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"scrollX":true,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"className":"dt-right","targets":[1,2,3,5,6,7,8,9,10,12,13,14,15,16,19]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<div class="figure" style="text-align: center"><span id="fig:plotMA-male-edgeR-fig1-m10"></span>
<img src="SLX-16110-16111_files/figure-html/plotMA-male-edgeR-fig1-m10-1.png" alt="MA plot (edgeR)" width="672" />
<p class="caption">
Figure 6.7: MA plot (edgeR)
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:plotVol-male-edgeR-fig1-m10"></span>
<img src="SLX-16110-16111_files/figure-html/plotVol-male-edgeR-fig1-m10-1.png" alt="Volcano plot (edgeR)" width="672" />
<p class="caption">
Figure 6.8: Volcano plot (edgeR)
</p>
</div>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="diffbind.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="chipseqspikeinfree.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["SLX-16110-16111.pdf", "SLX-16110-16111.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
