<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | SLX-16110-16111 Run Report (ChIP-Seq)</title>
  <meta name="description" content="This is a report for Irving Aye’s `r params$seq_type’ results" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="References | SLX-16110-16111 Run Report (ChIP-Seq)" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a report for Irving Aye’s `r params$seq_type’ results" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | SLX-16110-16111 Run Report (ChIP-Seq)" />
  
  <meta name="twitter:description" content="This is a report for Irving Aye’s `r params$seq_type’ results" />
  



<meta name="date" content="2021-02-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="chipseqspikeinfree.html"/>

<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-3.5.1/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.17/datatables.js"></script>
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<script src="libs/jszip-1.10.20/jszip.min.js"></script>
<link href="libs/dt-ext-buttons-1.10.20/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="libs/dt-ext-buttons-1.10.20/js/dataTables.buttons.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.flash.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.html5.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.colVis.min.js"></script>
<script src="libs/dt-ext-buttons-1.10.20/js/buttons.print.min.js"></script>
<link href="libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Run Report</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Samples</a></li>
<li class="chapter" data-level="2" data-path="mapping.html"><a href="mapping.html"><i class="fa fa-check"></i><b>2</b> Mapping</a>
<ul>
<li class="chapter" data-level="2.1" data-path="mapping.html"><a href="mapping.html#table-of-mapping-statistic"><i class="fa fa-check"></i><b>2.1</b> Table of mapping statistic</a></li>
<li class="chapter" data-level="2.2" data-path="mapping.html"><a href="mapping.html#boxplots-of-mapping-staststic"><i class="fa fa-check"></i><b>2.2</b> Boxplots of mapping staststic</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="mapping.html"><a href="mapping.html#no.-of-sequencing-reads"><i class="fa fa-check"></i><b>2.2.1</b> No. of sequencing reads</a></li>
<li class="chapter" data-level="2.2.2" data-path="mapping.html"><a href="mapping.html#no.-of-uniquely-mapped-reads"><i class="fa fa-check"></i><b>2.2.2</b> No. of uniquely mapped reads</a></li>
<li class="chapter" data-level="2.2.3" data-path="mapping.html"><a href="mapping.html#unique-mapping-rate"><i class="fa fa-check"></i><b>2.2.3</b> Unique mapping rate</a></li>
<li class="chapter" data-level="2.2.4" data-path="mapping.html"><a href="mapping.html#multi-mapping-rate"><i class="fa fa-check"></i><b>2.2.4</b> Multi mapping rate</a></li>
<li class="chapter" data-level="2.2.5" data-path="mapping.html"><a href="mapping.html#total-mapping-rate"><i class="fa fa-check"></i><b>2.2.5</b> Total mapping rate</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="mapping.html"><a href="mapping.html#bigwig-files"><i class="fa fa-check"></i><b>2.3</b> BigWig files</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="calling.html"><a href="calling.html"><i class="fa fa-check"></i><b>3</b> Peak Calling</a>
<ul>
<li class="chapter" data-level="3.1" data-path="calling.html"><a href="calling.html#no.-of-peaks"><i class="fa fa-check"></i><b>3.1</b> No. of peaks</a></li>
<li class="chapter" data-level="3.2" data-path="calling.html"><a href="calling.html#no.-of-peaks-by-the-consensus-level"><i class="fa fa-check"></i><b>3.2</b> No. of peaks by the consensus level</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="calling.html"><a href="calling.html#all-samples-n22"><i class="fa fa-check"></i><b>3.2.1</b> All samples (n=22)</a></li>
<li class="chapter" data-level="3.2.2" data-path="calling.html"><a href="calling.html#fv-n5"><i class="fa fa-check"></i><b>3.2.2</b> FV (n=5)</a></li>
<li class="chapter" data-level="3.2.3" data-path="calling.html"><a href="calling.html#fd-n5"><i class="fa fa-check"></i><b>3.2.3</b> FD (n=5)</a></li>
<li class="chapter" data-level="3.2.4" data-path="calling.html"><a href="calling.html#mv-n6"><i class="fa fa-check"></i><b>3.2.4</b> MV (n=6)</a></li>
<li class="chapter" data-level="3.2.5" data-path="calling.html"><a href="calling.html#md-n6"><i class="fa fa-check"></i><b>3.2.5</b> MD (n=6)</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="calling.html"><a href="calling.html#consensus_peak"><i class="fa fa-check"></i><b>3.3</b> Consensus peaks across factors of interests</a></li>
<li class="chapter" data-level="3.4" data-path="calling.html"><a href="calling.html#fraction-of-read-in-peaks-frip"><i class="fa fa-check"></i><b>3.4</b> Fraction of Read in Peaks (FRiP)</a></li>
<li class="chapter" data-level="3.5" data-path="calling.html"><a href="calling.html#heatmap"><i class="fa fa-check"></i><b>3.5</b> Heatmap</a></li>
<li class="chapter" data-level="3.6" data-path="calling.html"><a href="calling.html#pca"><i class="fa fa-check"></i><b>3.6</b> PCA</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="peak-annotations.html"><a href="peak-annotations.html"><i class="fa fa-check"></i><b>4</b> Peak Annotations</a>
<ul>
<li class="chapter" data-level="4.1" data-path="peak-annotations.html"><a href="peak-annotations.html#no.-of-peaks-by-group"><i class="fa fa-check"></i><b>4.1</b> No. of peaks by group</a></li>
<li class="chapter" data-level="4.2" data-path="peak-annotations.html"><a href="peak-annotations.html#no.-of-peaks-by-chromosome"><i class="fa fa-check"></i><b>4.2</b> No. of peaks by chromosome</a></li>
<li class="chapter" data-level="4.3" data-path="peak-annotations.html"><a href="peak-annotations.html#no.-of-genes-enrichred-in-peaks"><i class="fa fa-check"></i><b>4.3</b> No. of genes enrichred in peaks</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="peak-annotations.html"><a href="peak-annotations.html#non-intergenic-peaks-only"><i class="fa fa-check"></i><b>4.3.1</b> Non-intergenic peaks only</a></li>
<li class="chapter" data-level="4.3.2" data-path="peak-annotations.html"><a href="peak-annotations.html#peaks-in-tss-2kb"><i class="fa fa-check"></i><b>4.3.2</b> Peaks in TSS (&lt;2kb)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="peak-annotations.html"><a href="peak-annotations.html#genomic-feature-of-peaks"><i class="fa fa-check"></i><b>4.4</b> Genomic feature of peaks</a></li>
<li class="chapter" data-level="4.5" data-path="peak-annotations.html"><a href="peak-annotations.html#distance-to-tss"><i class="fa fa-check"></i><b>4.5</b> Distance to TSS</a></li>
<li class="chapter" data-level="4.6" data-path="peak-annotations.html"><a href="peak-annotations.html#peak-frequency-over-tss"><i class="fa fa-check"></i><b>4.6</b> Peak frequency over TSS</a></li>
<li class="chapter" data-level="4.7" data-path="peak-annotations.html"><a href="peak-annotations.html#peak-heatmap-over-tss"><i class="fa fa-check"></i><b>4.7</b> Peak heatmap over TSS</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="diffbind.html"><a href="diffbind.html"><i class="fa fa-check"></i><b>5</b> Differential Binding Region</a>
<ul>
<li class="chapter" data-level="5.1" data-path="diffbind.html"><a href="diffbind.html#no.-of-dbr"><i class="fa fa-check"></i><b>5.1</b> No. of DBR</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="diffbind.html"><a href="diffbind.html#adjutedd-p-val0.05"><i class="fa fa-check"></i><b>5.1.1</b> adjutedd P-val&lt;0.05</a></li>
<li class="chapter" data-level="5.1.2" data-path="diffbind.html"><a href="diffbind.html#adjutedd-p-val0.01"><i class="fa fa-check"></i><b>5.1.2</b> adjutedd P-val&lt;0.01:</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="diffbind.html"><a href="diffbind.html#dbr-from-deseq2"><i class="fa fa-check"></i><b>5.2</b> DBR from DESeq2</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="diffbind.html"><a href="diffbind.html#normalisation-factors"><i class="fa fa-check"></i><b>5.2.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="5.2.2" data-path="diffbind.html"><a href="diffbind.html#male-md-vs-mv"><i class="fa fa-check"></i><b>5.2.2</b> Male (MD vs MV)</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="diffbind.html"><a href="diffbind.html#dbr-from-edger"><i class="fa fa-check"></i><b>5.3</b> DBR from edgeR</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="diffbind.html"><a href="diffbind.html#normalisation-factors-1"><i class="fa fa-check"></i><b>5.3.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="5.3.2" data-path="diffbind.html"><a href="diffbind.html#male-md-vs-mv-1"><i class="fa fa-check"></i><b>5.3.2</b> Male (MD vs MV)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html"><i class="fa fa-check"></i><b>6</b> Differential Binding Region (MD vs MV)</a>
<ul>
<li class="chapter" data-level="6.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#no.-of-dbr-1"><i class="fa fa-check"></i><b>6.1</b> No. of DBR</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#adjutedd-p-val0.05-1"><i class="fa fa-check"></i><b>6.1.1</b> adjutedd P-val&lt;0.05</a></li>
<li class="chapter" data-level="6.1.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#adjutedd-p-val0.01-1"><i class="fa fa-check"></i><b>6.1.2</b> adjutedd P-val&lt;0.01:</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#dbr-from-deseq2-1"><i class="fa fa-check"></i><b>6.2</b> DBR from DESeq2</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#normalisation-factors-2"><i class="fa fa-check"></i><b>6.2.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="6.2.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#male-md-vs-mv-2"><i class="fa fa-check"></i><b>6.2.2</b> Male (MD vs MV)</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#dbr-from-edger-1"><i class="fa fa-check"></i><b>6.3</b> DBR from edgeR</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#normalisation-factors-3"><i class="fa fa-check"></i><b>6.3.1</b> Normalisation factors</a></li>
<li class="chapter" data-level="6.3.2" data-path="diffbind-md-vs-mv-n-10.html"><a href="diffbind-md-vs-mv-n-10.html#male-md-vs-mv-3"><i class="fa fa-check"></i><b>6.3.2</b> Male (MD vs MV)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="chipseqspikeinfree.html"><a href="chipseqspikeinfree.html"><i class="fa fa-check"></i><b>7</b> ChIPseqSpikeInFree</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a>
<ul>
<li class="chapter" data-level="7.1" data-path="references.html"><a href="references.html#sessioninfo"><i class="fa fa-check"></i><b>7.1</b> sessionInfo</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">SLX-16110-16111 Run Report (ChIP-Seq)</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="sessioninfo" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> sessionInfo</h2>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="references.html#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>## R version 4.0.3 (2020-10-10)
## Platform: x86_64-conda-linux-gnu (64-bit)
## Running under: Scientific Linux 7.9 (Nitrogen)
## 
## Matrix products: default
## BLAS/LAPACK: /home/ssg29/miniconda3/envs/bioc3.12/lib/libopenblasp-r0.3.12.so
## 
## locale:
##  [1] LC_CTYPE=en_GB.UTF-8       LC_NUMERIC=C              
##  [3] LC_TIME=en_GB.UTF-8        LC_COLLATE=en_GB.UTF-8    
##  [5] LC_MONETARY=en_GB.UTF-8    LC_MESSAGES=en_GB.UTF-8   
##  [7] LC_PAPER=en_GB.UTF-8       LC_NAME=C                 
##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
## [11] LC_MEASUREMENT=en_GB.UTF-8 LC_IDENTIFICATION=C       
## 
## attached base packages:
## [1] parallel  stats4    stats     graphics  grDevices utils     datasets 
## [8] methods   base     
## 
## other attached packages:
##  [1] org.Hs.eg.db_3.12.0                     
##  [2] TxDb.Hsapiens.UCSC.hg38.knownGene_3.10.0
##  [3] GenomicFeatures_1.42.1                  
##  [4] AnnotationDbi_1.52.0                    
##  [5] ReactomePA_1.34.0                       
##  [6] clusterProfiler_3.18.0                  
##  [7] ChIPseeker_1.26.0                       
##  [8] ggplot2_3.3.3                           
##  [9] kableExtra_1.3.1                        
## [10] DiffBind_3.0.11                         
## [11] SummarizedExperiment_1.20.0             
## [12] Biobase_2.50.0                          
## [13] MatrixGenerics_1.2.0                    
## [14] matrixStats_0.57.0                      
## [15] GenomicRanges_1.42.0                    
## [16] GenomeInfoDb_1.26.2                     
## [17] IRanges_2.24.1                          
## [18] S4Vectors_0.28.1                        
## [19] BiocGenerics_0.36.0                     
## [20] BiocParallel_1.24.1                     
## [21] yaml_2.2.1                              
## [22] nvimcom_0.9-106                         
## [23] data.table_1.13.6                       
## [24] colorout_1.2-2                          
## 
## loaded via a namespace (and not attached):
##   [1] tidyselect_1.1.0                       
##   [2] htmlwidgets_1.5.3                      
##   [3] RSQLite_2.2.3                          
##   [4] grid_4.0.3                             
##   [5] scatterpie_0.1.5                       
##   [6] munsell_0.5.0                          
##   [7] base64url_1.4                          
##   [8] DT_0.17                                
##   [9] systemPipeR_1.24.3                     
##  [10] withr_2.4.0                            
##  [11] colorspace_2.0-0                       
##  [12] GOSemSim_2.16.1                        
##  [13] Category_2.56.0                        
##  [14] highr_0.8                              
##  [15] knitr_1.30                             
##  [16] rstudioapi_0.13                        
##  [17] ggsignif_0.6.0                         
##  [18] DOSE_3.16.0                            
##  [19] labeling_0.4.2                         
##  [20] bbmle_1.0.23.1                         
##  [21] GenomeInfoDbData_1.2.4                 
##  [22] mixsqp_0.3-43                          
##  [23] hwriter_1.3.2                          
##  [24] polyclip_1.10-0                        
##  [25] bit64_4.0.5                            
##  [26] farver_2.0.3                           
##  [27] pheatmap_1.0.12                        
##  [28] downloader_0.4                         
##  [29] coda_0.19-4                            
##  [30] batchtools_0.9.15                      
##  [31] vctrs_0.3.6                            
##  [32] generics_0.1.0                         
##  [33] xfun_0.20                              
##  [34] csaw_1.24.3                            
##  [35] BiocFileCache_1.14.0                   
##  [36] R6_2.5.0                               
##  [37] apeglm_1.12.0                          
##  [38] graphlayouts_0.7.1                     
##  [39] invgamma_1.1                           
##  [40] locfit_1.5-9.4                         
##  [41] rsvg_2.1                               
##  [42] bitops_1.0-6                           
##  [43] fgsea_1.16.0                           
##  [44] DelayedArray_0.16.1                    
##  [45] assertthat_0.2.1                       
##  [46] scales_1.1.1                           
##  [47] ggraph_2.0.4                           
##  [48] enrichplot_1.10.1                      
##  [49] gtable_0.3.0                           
##  [50] tidygraph_1.2.0                        
##  [51] rlang_0.4.10                           
##  [52] genefilter_1.72.1                      
##  [53] splines_4.0.3                          
##  [54] rtracklayer_1.50.0                     
##  [55] brew_1.0-6                             
##  [56] checkmate_2.0.0                        
##  [57] BiocManager_1.30.10                    
##  [58] reshape2_1.4.4                         
##  [59] crosstalk_1.1.1                        
##  [60] backports_1.2.1                        
##  [61] qvalue_2.22.0                          
##  [62] RBGL_1.66.0                            
##  [63] tools_4.0.3                            
##  [64] bookdown_0.21                          
##  [65] ellipsis_0.3.1                         
##  [66] gplots_3.1.1                           
##  [67] RColorBrewer_1.1-2                     
##  [68] Rcpp_1.0.6                             
##  [69] plyr_1.8.6                             
##  [70] progress_1.2.2                         
##  [71] zlibbioc_1.36.0                        
##  [72] purrr_0.3.4                            
##  [73] RCurl_1.98-1.2                         
##  [74] prettyunits_1.1.1                      
##  [75] openssl_1.4.3                          
##  [76] viridis_0.5.1                          
##  [77] cowplot_1.1.1                          
##  [78] ashr_2.2-47                            
##  [79] ggrepel_0.9.1                          
##  [80] magrittr_2.0.1                         
##  [81] DO.db_2.9                              
##  [82] reactome.db_1.74.0                     
##  [83] truncnorm_1.0-8                        
##  [84] mvtnorm_1.1-1                          
##  [85] SQUAREM_2021.1                         
##  [86] amap_0.8-18                            
##  [87] hms_1.0.0                              
##  [88] evaluate_0.14                          
##  [89] xtable_1.8-4                           
##  [90] XML_3.99-0.5                           
##  [91] emdbook_1.3.12                         
##  [92] jpeg_0.1-8.1                           
##  [93] gridExtra_2.3                          
##  [94] compiler_4.0.3                         
##  [95] biomaRt_2.46.1                         
##  [96] bdsmatrix_1.3-4                        
##  [97] tibble_3.0.5                           
##  [98] shadowtext_0.0.7                       
##  [99] KernSmooth_2.23-18                     
## [100] V8_3.4.0                               
## [101] crayon_1.3.4                           
## [102] htmltools_0.5.1.1                      
## [103] GOstats_2.56.0                         
## [104] geneplotter_1.68.0                     
## [105] tidyr_1.1.2                            
## [106] DBI_1.1.1                              
## [107] tweenr_1.0.1                           
## [108] dbplyr_2.0.0                           
## [109] MASS_7.3-53                            
## [110] rappdirs_0.3.1                         
## [111] boot_1.3-26                            
## [112] ShortRead_1.48.0                       
## [113] Matrix_1.3-2                           
## [114] igraph_1.2.6                           
## [115] pkgconfig_2.0.3                        
## [116] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
## [117] rvcheck_0.1.8                          
## [118] GenomicAlignments_1.26.0               
## [119] numDeriv_2016.8-1.1                    
## [120] xml2_1.3.2                             
## [121] annotate_1.68.0                        
## [122] webshot_0.5.2                          
## [123] XVector_0.30.0                         
## [124] AnnotationForge_1.32.0                 
## [125] rvest_0.3.6                            
## [126] stringr_1.4.0                          
## [127] VariantAnnotation_1.36.0               
## [128] digest_0.6.27                          
## [129] graph_1.68.0                           
## [130] Biostrings_2.58.0                      
## [131] rmarkdown_2.6                          
## [132] fastmatch_1.1-0                        
## [133] edgeR_3.32.1                           
## [134] GSEABase_1.52.1                        
## [135] GreyListChIP_1.22.0                    
## [136] curl_4.3                               
## [137] graphite_1.36.0                        
## [138] Rsamtools_2.6.0                        
## [139] gtools_3.8.2                           
## [140] rjson_0.2.20                           
## [141] lifecycle_0.2.0                        
## [142] jsonlite_1.7.2                         
## [143] viridisLite_0.3.0                      
## [144] askpass_1.1                            
## [145] limma_3.46.0                           
## [146] BSgenome_1.58.0                        
## [147] pillar_1.4.7                           
## [148] lattice_0.20-41                        
## [149] plotrix_3.8-1                          
## [150] httr_1.4.2                             
## [151] survival_3.2-7                         
## [152] GO.db_3.12.1                           
## [153] glue_1.4.2                             
## [154] png_0.1-7                              
## [155] bit_4.0.4                              
## [156] Rgraphviz_2.34.0                       
## [157] ggforce_0.3.2                          
## [158] stringi_1.5.3                          
## [159] blob_1.2.1                             
## [160] DESeq2_1.30.0                          
## [161] latticeExtra_0.6-29                    
## [162] caTools_1.18.1                         
## [163] memoise_1.1.0                          
## [164] DOT_0.1                                
## [165] dplyr_1.0.3                            
## [166] irlba_2.3.3</code></pre>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="chipseqspikeinfree.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["SLX-16110-16111.pdf", "SLX-16110-16111.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
